src/main/cli/cli.c

```diff
@@ -4829,6 +4829,10 @@ if (buildKey) {
     cliPrintLinefeed();
 #endif // USE_GPS
 
+#ifdef LOCALE
+    cliPrintLinef("%s %s", STR_LOCALE_SETUP, STR_LOCALE);
```

Comment:
```suggestion
    cliPrintLinef("Language: %s", STR_LOCALE_SETUP, STR_LOCALE);
```

Reply:
You are right, consequence of no translation in cli.c, Thx
changed to: cliPrintLinef("Language: %s", STR_LOCALE);